dali/test/python/test_detection_pipeline.py

```diff
@@ -332,7 +340,7 @@ def run_for_dataset(args, dataset):
                 image_flipped_cpu, image_flipped_gpu,\
                 boxes_flipped_cpu, boxes_flipped_gpu, \
                 encoded_boxes_cpu, encoded_boxes_gpu, \
-                encoded_labels_cpu, encoded_labels_gpu = \
+                encoded_labels_cpu, encoded_labels_gpu, \
```

Comment:
Maybe some typo here? I don't think it works like this.

Reply:
My bad

Reply:
Done